packages/backend-common/src/reading/integration.test.ts

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ConfigReader } from '@backstage/config';
+import { getVoidLogger } from '../logging';
+import { UrlReaders } from './UrlReaders';
+
+const reader = UrlReaders.default({
+  logger: getVoidLogger(),
+  config: new ConfigReader({
+    // The tokens in this config provide read only access to the backstage-verification repos
+    integrations: {
+      github: [
+        {
+          host: 'github.com',
```

Comment:
No token here may cause the rate limit to be hit very often. 60 requests per hour per IP address. 🤞 

However, the good news is since we are running the test in GitHub Actions CI, we already have a `env.GITHUB_TOKEN` which we can use here!

Reply:
`env.GITHUB_TOKEN` is better than `env.GH_BACKSTAGE_SERVICE_ACCOUNT...` since the later one is not shared on PRs created from forks.

Reply:
@freben had a nice hax for being able to set up another GitHub account for us, so this PR now also includes a github token